.github/workflows/ci-cd.yml

```diff
@@ -0,0 +1,119 @@
+# ci-cd.yml (Backend)
+name: Backend CI/CD Pipeline
+
+on:
+  push:
+    branches: [ main ]
+
+
+env:
+  DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+  DOCKER_PASSWORD: ${{ secrets.DOCKERHUB_TOKEN }}
+  AWS_REGION: ${{ secrets.AWS_REGION }}
+  K8S_NAMESPACE: ${{ secrets.K8S_NAMESPACE }}
+  EKS_CLUSTER_NAME: ${{ secrets.EKS_CLUSTER_NAME }}
+  # AWS_REGION: 'us-east-1' #CHANGE THESE TO SECRETS AFTER CONFIRMING THAT THEY WORK
+  # K8S_NAMESPACE: 'shredder' #CHANGE THESE TO SECRETS AFTER CONFIRMING THAT THEY WORK
+  # EKS_CLUSTER_NAME: 'shredder_cluster' #CHANGE THESE TO SECRETS AFTER CONFIRMING THAT THEY WORK
+  
+jobs:
+  backend-test:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:

```

Comment:
I think this might cause issues. The purpose of this job is to make sure the latest Dockerfile changes are pushed to dockerhub and if we made changes while the database was down it would push those changes up. For instance, if we wanted to test before having the rest of the resources stood up. 

I could be wrong though.